packet/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -3,35 +3,15 @@ resource "packet_device" "nodes" {
   hostname         = "${var.cluster_name}-${var.pool_name}-worker-${count.index}"
   plan             = "${var.type}"
   facilities       = ["${var.facility}"]
-  operating_system = "custom_ipxe"
+  operating_system = "flatcar_${var.os_channel}"
   billing_cycle    = "hourly"
   project_id       = "${var.project_id}"
-  ipxe_script_url  = "${var.ipxe_script_url}"
```

Comment:
This variable can be removed from the variables.tf. It is not being used anymore, IIUC

Reply:
Done.